rmw/include/rmw/qos_profiles.h

```diff
@@ -22,15 +22,20 @@ extern "C"
 
 #include "rmw/types.h"
 
+#define rmw_qos_deadline_default {0, 0}
```

Comment:
I would prefer `{0}` because it is more type-agnostic, but this is a minor issue.

I do worry about the Windows builds when using `struct timespec`.

Reply:
Is windows not C11-compliant?

Reply:
Oh! I found rmw_time_t - I will change to use that

Reply:
Not sure, but Microsoft is pretty notorious about neglecting C, and telling their customers to just use C++.